app/views/users/_dashboard_deck.html.erb

```diff
@@ -16,7 +16,19 @@
       <div class="count">
         <%= pluralize(deck.cards.count, 'Card') %>
       </div>
+      <div class="difficulty">
+        9.5 Difficulty
+      </div>
     </div>
   </div>
+  <button 
+    type="button"
+    class="btn btn-preview"
+    data-toggle="popover"
+    data-trigger="click"
+    data-original-title="<div>Content Preview</div>"
+    data-html="true" data-content="<%= deck.cards.offset(rand(deck.cards.count)).first.content %>">
```

Comment:
We will probably want to add a column on the `decks` table for description and we can show that content here. I don't know how meaningful it is to preview the content of one card. If we go with description we'd also want to change any text that says `preview` to `description`